src/main.rs

```diff
@@ -35,6 +35,8 @@ mod config;
 mod git;
 mod lei;
 
+type BoxPath = Box<Path>;
```

Comment:
any reason for not using `PathBuf`?

Reply:
I just tend to use boxes for immutable types to save the pointer, but this is just for an error so I'll drop it